src/System.Composition.Convention/src/System/Composition/Convention/ConventionBuilder.cs

```diff
@@ -340,22 +340,22 @@ private List<Attribute> ReadParameterCustomAttributes(Type reflectedType, System
             return cachedAttributes;
         }
 
-        private static bool IsGenericDescendentOf(TypeInfo openType, TypeInfo baseType)
+        private static bool IsGenericDescendentOf(TypeInfo descendentType, TypeInfo baseType)
```

Comment:
Is the rename because the type you are passing in could be a non-generic type? 

If we are going to rename it, how about we call it `derivedType` to match the other parameter?